tokendito/helpers.py

```diff
@@ -312,6 +312,25 @@ def validate_saml_response(html):
     return xml
 
 
+def validate_okta_aws_app_url(url):
+    """Validate whether a given URL is a valid AWS app URL in Okta.
+
+    :param url: string
+    :return: bool. True if valid, False otherwise
+    """
+    logging.debug('Will try to match \'{}\' to a valid URL'.format(url))
+
+    url = urlparse(url)
```

Comment:
use a new local var name since url is a parameter

Reply:
fixed.